core/templates/dev/head/pages/exploration_editor/editor_tab/StateHints.js

```diff
@@ -51,6 +51,13 @@ oppia.controller('StateHints', [
     };
 
     $scope.changeActiveHintIndex = function(newIndex) {
+      var currentActiveIndex = $scope.activeHintIndex;
+      if (Number.isInteger(currentActiveIndex) && (
```

Comment:
Semantically, should this actually be something like `currentActiveIndex !== null`?